woowacrew-domain/src/test/java/woowacrew/github/domain/GithubCommitTest.java

```diff
@@ -28,4 +29,14 @@
             new GithubCommit(mockUser, date, 300);
         });
     }
+
+    @Test
+    void 같은_유저인지_확인한다() {
+        User mockUser = mock(User.class);
+        GithubCommit githubCommit = new GithubCommit(mockUser, LocalDate.of(2020, 6, 1), 300);
+
+        when(mockUser.isSameUser(any())).thenReturn(true);
```

Comment:
```suggestion
        when(mockUser.isSameUser(any(User.class))).thenReturn(true);
```

any() 대신 primitive type의 경우엔 anyChar() 같은 계열, 객체들은 isA(Class) 또는 any(Class)를 사용하는게 좋을 거 같습니다.(For primitive types use anyChar() family or isA(Class) or any(Class))

Reply:
좋은 피드백 감사합니다!